Source/Core/DolphinQt/Achievements/AchievementHeaderWidget.cpp

```diff
@@ -157,6 +160,12 @@ void AchievementHeaderWidget::UpdateData()
     m_game_progress_hard->setVisible(false);
     m_game_progress_soft->setVisible(false);
     m_rich_presence->setVisible(false);
+    if (AchievementManager::GetInstance()->IsDisabled())
+    {
+      m_locked_warning->setText(tr("Achievements have been disabled.<br>Please close all running "
+                                   "games to re-enable achievements."));
+      m_locked_warning->setStyleSheet(QStringLiteral("QLabel { color : red; }"));
```

Comment:
This is never set to anything else, you can just init the text once and then call setVisible(true/false) depending on if it's disabled or not. In fact I suspect your current approach doesn't work correctly because the label is never set visible again once it's hidden.